bugbot/rules/workflow/no_severity_ni.py

```diff
@@ -193,6 +201,25 @@ def get_bz_params(self, date):
 
         return params
 
+    def filter_bugs(self, bugs):
+        users_info = UserActivity(include_fields=["groups", "requests"]).check_users(
+            set(bug["triage_owner"] for bug in bugs.values()),
+            keep_active=True,
+            fetch_employee_info=True,
+        )
+
+        for bug_id, bug in list(bugs.items()):
+            user_info = users_info[bug["triage_owner"]]
+            if "requests" in user_info:
+                if user_info["requests"]["needinfo"]["blocked"]:
+                    del bugs[bug_id]
+        return bugs
```

Comment:
I do not think we should filter out the bugs; we should only avoid the needinfo. Also, we should decide if we want to try to find someone else.